app/assets/stylesheets/pages.scss

```diff
@@ -24,4 +24,8 @@ tr.shown td.details-control {
   bottom: 0;
   right: 0;
   height: auto;
-}
\ No newline at end of file
+}
+
+.div-inline {
```

Comment:
Not necessary if you use Bootstrap documented way of making button dropdowns (see comment below).